pom.xml

```diff
@@ -474,7 +479,30 @@
         <artifactId>maven-release-plugin</artifactId>
         <version>2.5.3</version>
       </plugin>
+
+	  <!-- For all subprojects, read a top-level local.properties file for configuration 
+	       information. This can override the properties needed to, e.g., connect to a DB. -->
```

Comment:
So is this `local.properties` parsed at runtime? Or is it more to override some of the things in `default.properties`, the advantage being that it is more clear exactly what is unique to the local configuration.